### ISSUE TITLE
Circle CI / Semantic Release: Authenticity of host github.com can't be established.

### ISSUE THREAD

**Issue by BBlackwo** [btroncone/ngrx-store-localstorage#192]
The semantic release deploy step has been failing for a few weeks now with the following error:

```sh
[5:07:02 AM] [semantic-release] › ✔  Loaded plugin "fail" from "@semantic-release/github"
The authenticity of host 'github.com (140.82.112.3)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
```

**Comment by BBlackwo** [comment]
Attempts to fix:
- https://github.com/btroncone/ngrx-store-localstorage/pull/191
- https://github.com/btroncone/ngrx-store-localstorage/pull/190

**Comment by BBlackwo** [comment]
Circle CI Discuss Links:
- https://discuss.circleci.com/t/the-authenticity-of-github-host-cant-be-stablished/33133
- https://discuss.circleci.com/t/the-authenticity-of-host-github-com-cant-be-established/20626
- https://discuss.circleci.com/t/authenticity-of-host-cant-be-established-are-you-sure-you-want-to-continue-connecting-yes-no/25754

Issue Links:
- https://github.com/cypress-io/circleci-orb/issues/101
- https://github.com/semantic-release/semantic-release/issues/1095

Circle CI Docs:
- https://circleci.com/docs/2.0/add-ssh-key/
- https://circleci.com/docs/2.0/gh-bb-integration/#establishing-the-authenticity-of-an-ssh-host

**Comment by BBlackwo** [comment]
:tada: Fixed with #193! :tada:
